app/controllers/admin/settings/display_settings_controller.rb

```diff
@@ -52,7 +52,11 @@ def validate_start_of_week_year
       start_of_year = params[:settings][:first_week_of_year]
 
       if start_of_week.present? ^ start_of_year.present?
-        flash[:error] = I18n.t('settings.display.first_date_of_week_and_year_present')
+        flash[:error] = I18n.t(
+          'settings.display.first_date_of_week_and_year_present',
```

Comment:
If we want to force this to work in every l10n we'd have to rename the key to invalidate the already existing translations that will not make use of the included translations.

Reply:
Good point, however I believe crowdin should invalidate translations for changed source strings. In our `crowdin/de.yml` file however, they will be invalid until updated from crowdin then. I changed it